server/build/webpack.js

```diff
@@ -299,10 +299,11 @@ export default async function createCompiler (dir, { dev = false, quiet = false,
   }
 
   if (!dev) {
+    // We do this to use the minified version of React in production.
+    // This will significant file size redution when turned off uglifyjs.
     webpackConfig.resolve.alias = {
       'react': require.resolve('react/dist/react.min.js'),
-      'react-dom': require.resolve('react-dom/dist/react-dom.min.js'),
-      'react-dom/server': require.resolve('react-dom/dist/react-dom-server.min.js')
```

Comment:
Hmm.. how are you getting that error? I just pulled a fresh `master`, added that change, built, then copied to `examples/hello-world` once again. Built then ran that demo & works without errors.